src/features/courses/course-controller.ts

```diff
@@ -2381,6 +2382,7 @@ class CourseController {
                 [sequelize.fn('avg', sequelize.col(`topics.questions.grades.${StudentGrade.rawAttributes.numAttempts.field}`)), 'averageAttemptedCount'],
                 [averageScoreAttribute, 'averageScore'],
                 [sequelize.fn('count', sequelize.col(`topics.questions.grades.${StudentGrade.rawAttributes.id.field}`)), 'totalGrades'],
+                [sequelize.fn('avg', sequelize.col(`topics.questions.grades.${StudentGrade.rawAttributes.overallBestScore.field}`)), 'systemScore'],
```

Comment:
overallBestScore is not what we want -- the 'system score' should come from the partial credit best score...

Reply:
Good catches, I thought I fixed those.